### PR TITLE
Increment versions for sql releases

### PR DESCRIPTION
**Pull request by v-hongli1**
Only added change log of `2.34.0-beta.1` in `CHANGELOG.md`.

# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/resourcemanager/azure-resourcemanager-sql/CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,10 @@
 
 - Updated core dependency from resources.
 
+## 2.34.0-beta.1 (2023-12-22)
+
+- Azure Resource Manager Sql client library for Java. This package contains Microsoft Azure SDK for Sql Management SDK. Package tag package-composite-v5. For documentation on how to use this package, please see [Azure Management Libraries for Java](https://aka.ms/azsdk/java/mgmt).
```
Comment (XiaofeiCao) [inline sdk/resourcemanager/azure-resourcemanager-sql/CHANGELOG.md:23]:
```suggestion
- Preview release for `api-version` `2023-05-01-preview`.
```

Comment (v-hongli1) [reply sdk/resourcemanager/azure-resourcemanager-sql/CHANGELOG.md]: Fixed in the new version.

File: sdk/resourcemanager/azure-resourcemanager-sql/CHANGELOG.md [hunk 2]
```diff
@@ -18,6 +18,10 @@
 
 - Updated core dependency from resources.
 
+## 2.34.0-beta.1 (2023-12-22)
```
Comment (XiaofeiCao) [inline sdk/resourcemanager/azure-resourcemanager-sql/CHANGELOG.md:21]:
```suggestion
## 2.34.0-beta.1 (2023-12-25)
```

Comment (v-hongli1) [reply sdk/resourcemanager/azure-resourcemanager-sql/CHANGELOG.md]: Fixed in the new version.

**Review by XiaofeiCao — APPROVED** [review]

**Pull request merged**